src/main/kotlin/app/cash/licensee/licenses.kt

```diff
@@ -110,6 +110,16 @@ private fun PomLicense.toSpdx(): List<SpdxLicense> {
       "https://www.gnu.org/software/classpath/license.html",
       -> "GPL-2.0-with-classpath-exception"
 
+      "http://choosealicense.com/licenses/gpl-2.0",
+      "https://choosealicense.com/licenses/gpl-2.0",
+      "http://opensource.org/license/gpl-2-0",
+      "https://opensource.org/license/gpl-2-0",
+      "http://www.gnu.org/licenses/old-licenses/gpl-2.0.html",
+      "https://www.gnu.org/licenses/old-licenses/gpl-2.0.html",
+      "http://api.github.com/licenses/gpl-2.0",
+      "https://api.github.com/licenses/gpl-2.0",
+      -> "GPL-2.0-or-later"
```

Comment:
I wonder if we should change this `when` to return a `List<String>` and then `map` those entries to the `SpdxLicense`. This way these URLs would match both `GPL-2.0-or-later` and `GPL-2.0-only`.

@hfhbd 

Can think about it for a follow-up.

Also the logic below should fail on a non-null fallback ID which fails to match a known SPDX identifier.